pykafka/protocol.py

```diff
@@ -169,6 +171,7 @@ class Message(Message, Serializable):
         "produce_attempt",
         "delivery_report_q",
         "protocol_version",
+        "compressed_offset",
```

Comment:
Naming: `compressed_offset` might be misconstrued as "an offset that was compressed". Consider renaming this to something like `offset_was_compressed`.